data/human/deep jobs.txt

```diff
@@ -353,3 +353,617 @@ mission "Record academic conference"
+
+
+phrase "stranded consultant pickup"
+	word
+		`In accordance with the lodging request, you`
+		`Despite the short notice, you`
+	word
+		` `
+	word
+		`meet the consultants at their hotel accommodation`
+		`collect the consultants from the security station`
+		`fish the consultants out of a seedy bar`
+		`rescue the consultants from some overbearing companions`
+	word
+		` `
+	word
+		`in a quiet part of the spaceport`
+		`on the other side of town`
+		`in a raucous quarter just outside the port`
+	word
+		` `
+	word
+		`and escort them on board.`
+
+phrase "stranded consultant dropoff payment"
+	word
+		`The consultants thank you as you deliver them to their destination.`
+	word
+		` `
+	word
+		`You find out only afterwards that they left behind a soiled pair of boots.`
+		`Afterwards you find a tie left hanging on a door handle.`
+		`Later, you find some crumpled documents left behind and wisely tip them into the ship's incinerator without looking at them.`
+	word
+		` `
+	word
+		`Your bank account soon notifies you that the agreed-upon payment of <payment> has been deposited.`
+
+
+mission "Transport Consultants (Deep Security) [0]"
+	name "Transport Deep Security consultants to <planet>"
+	job
+	repeat
+	description "Transport consultants whose schedules have been ruined by Deep Security to <destination>. Payment is <payment>."
+	passengers 1 8
+	to offer
+		random < 20
+		has "deep security consultants"
+		"reputation: Deep Security" >= 100
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		attributes "urban"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "stranded consultant pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 200000 35000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "stranded consultant dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 4
+	on abort
+		"reputation: Deep Security" -= 10
+
+mission "Transport Consultants (Deep Security) [1]"
+	name "Transport Deep Security consultants to <planet>"
+	job
+	repeat
+	description "Transport consultants whose schedules have been ruined by Deep Security to <destination>. Payment is <payment>."
+	passengers 1 8
+	to offer
+		random < 20
+		has "deep security consultants"
+		"reputation: Deep Security" >= 100
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		attributes "urban"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "stranded consultant pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 250000 40000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "stranded consultant dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 4
+	on abort
+		"reputation: Deep Security" -= 10
+
+
+mission "Urgent Deep Sec Package"
+	name "Rush delivery of a Deep Security package to <planet>"
```

Comment:
Some of these job names seem especially long. Do they display okay in-game?

Reply:
I honestly haven't tested. I reckon I could revise that to "Deliver a Deep Sec package to <planet>" which I'm fairly sure would fit.

Reply:
Could be shortened to something like `Rush Deep Security Delivery to <planet>`